D1673/P1673.md

```diff
@@ -6614,65 +6619,27 @@ where
 
     * else, `Accessor`.
 
-[3]{.pnum} *Effects:*
+[3]{.pnum} *Returns:*
 
-  * [3.1]{.pnum} If `Layout` is `layout_left`, `layout_right`, or
-      `layout_blas_packed<Triangle, StorageOrder>`
-      for some `Triangle` and `StorageOrder`,
-      then equivalent to
-```c++
-return R(a.data(),
-  ReturnMapping(@_transpose-extents_@(a.mapping().extents())),
-  a.accessor());
-```
+  * [3.1]{.pnum} `R(a.data(), ReturnMapping(@_transpose-extents_@(a.mapping().extents())),
+      a.accessor())` if `Layout` is `layout_left`, `layout_right`, or a specialization of
+      `layout_blas_packed`.
```

Comment:
I think breaking the line inside backquotes won't format correctly.  This also applies to the changes below.

```suggestion
  * [3.1]{.pnum} `R(a.data(), ReturnMapping(@_transpose-extents_@(a.mapping().extents())), a.accessor())`
      if `Layout` is `layout_left`, `layout_right`,
      or a specialization of `layout_blas_packed`.
```